ci/check_history_consistency.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+"""
+Checks the consistency of the history, by comparing flask db
+output and the filesystem ordering of files.
+"""
+import re
+import subprocess
+from pathlib import Path
+
+
+def _iter_db_revisions():
+    output = (
+        subprocess.check_output(['flask', 'db', 'history'], text=True)
+        .strip()
+        .split('\n')
+    )
+    pattern = re.compile(r'^.* -> ([a-zA-Z0-9]+)( \(head\))?,')
+    for line in output:
+        for obj in pattern.finditer(line):
+            yield obj.group(1)
+
+
+def _iter_file_revisions():
+    root_path = Path('newdle/migrations/versions')
+    pattern = re.compile(r"\srevision = '(.*)'")
+    for pyfile in sorted(root_path.glob('*.py')):
+        results = pattern.findall(pyfile.read_text())
+        yield results[0]
+
+
+def _check_history_consistency():
+    revisions = list(_iter_db_revisions())
+    file_revisions = list(_iter_file_revisions())
+    revisions.reverse()
+    assert revisions == file_revisions
```

Comment:
Tthe error in case of an inconsistency could be a bit more detailed ;)

Reply:
Raising an exception with more information now.